Sources/docc/DocCDocumentation.docc/adding-tables-of-data.md

```diff
@@ -0,0 +1,160 @@
+# Adding Tables of Data
+
+Arrange information into rows and columns.
+
+## Overview
+
+To add a table to your documentation, start a new paragraph and add the following elements, each on its own line:
+
+* A table header, with the heading text of each column, separated by pipes (`|`)
```

Comment:
Are table headers optional? Or must every table have a header row?

Reply:
No. The header, the separator row, and at least one row of content is required.

Reply:
I clarified that all these elements are required in https://github.com/apple/swift-docc/pull/813/commits/9a377728c9eea1de2f153a9a9bfa72ea97938afa